src/shaders/shadow.js

```diff
@@ -0,0 +1,29 @@
+var registerShader = require('../core/shader').registerShader;
+var THREE = require('../lib/three');
+
+/**
+ * Flat shader using THREE.ShadowMaterial.
+ */
+module.exports.Shader = registerShader('shadow', {
```

Comment:
Is this shader used?

Reply:
I just updated the sample to include a ground shadow in AR. This sample uses the special shadow shader to accomplish this as it can't be done using regular materials.